### ISSUE TITLE
Error: [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.

### ISSUE THREAD

**Issue by nicky-lenaers** [zefoy/ngx-perfect-scrollbar#198]
Hi,

When I use touch to scroll in Chrome, I get these error (they stack on every touch event):

```sh
[Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
perfect-scrollbar.esm.js:1041 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
touchMove @ perfect-scrollbar.esm.js:1041
push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask @ zone.js:421
push../node_modules/zone.js/dist/zone.js.Zone.runTask @ zone.js:188
push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask @ zone.js:496
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
perfect-scrollbar.esm.js:1041 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
touchMove @ perfect-scrollbar.esm.js:1041
push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask @ zone.js:421
push../node_modules/zone.js/dist/zone.js.Zone.runTask @ zone.js:188
push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask @ zone.js:496
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
perfect-scrollbar.esm.js:1041 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
touchMove @ perfect-scrollbar.esm.js:1041
push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask @ zone.js:421
push../node_modules/zone.js/dist/zone.js.Zone.runTask @ zone.js:188
push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask @ zone.js:496
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
perfect-scrollbar.esm.js:1041 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
touchMove @ perfect-scrollbar.esm.js:1041
push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask @ zone.js:421
push../node_modules/zone.js/dist/zone.js.Zone.runTask @ zone.js:188
push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask @ zone.js:496
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
perfect-scrollbar.esm.js:1041 [Intervention] Ignored attempt to cancel a touchmove event with cancelable=false, for example because scrolling is in progress and cannot be interrupted.
touchMove @ perfect-scrollbar.esm.js:1041
push../node_modules/zone.js/dist/zone.js.ZoneDelegate.invokeTask @ zone.js:421
push../node_modules/zone.js/dist/zone.js.Zone.runTask @ zone.js:188
push../node_modules/zone.js/dist/zone.js.ZoneTask.invokeTask @ zone.js:496
invokeTask @ zone.js:1540
globalZoneAwareCallback @ zone.js:1566
```

Using:
- Angular 6.1.x
- Chrome 67.0
- Node 10

**Comment by EightCanisters** [comment]
Has the problem been solved? I have the same problem.

**Comment by uzumakinaruto123** [comment]
This should be reopened. This occurs frequently on mobile devices and impacts other pages/DOM elements making them unusable,

**Comment by sconix** [comment]
This is not about this library, the error is in Perfect Scrollbar. On the side note we have never seen this in our apps and we use heavily this library.

**Comment by thiagoteberga** [comment]
Has the problem been solved? I have the same problem.

**Comment by thiagoteberga** [comment]
It seems that the problem was solved in version 1.5.3, however the project uses version 1.5.0.
https://github.com/mdbootstrap/Tailwind-Elements/pull/1017

**Comment by thiagoteberga** [comment]
I sent the PR:
https://github.com/zefoy/ngx-perfect-scrollbar/pull/343
